src/components/Login/LocalLogin.tsx

```diff
@@ -101,17 +107,23 @@ const LocalLogin: React.FC<LocalLoginProps> = ({ revalidate }) => {
                 )}
               </div>
               <div className="pt-5 mt-8 border-t border-gray-700">
-                <div className="flex justify-between">
-                  <span className="inline-flex rounded-md shadow-sm">
-                    <Link href="/resetpassword" passHref>
-                      <Button as="a" buttonType="ghost">
-                        <SupportIcon />
-                        <span>
-                          {intl.formatMessage(messages.forgotpassword)}
-                        </span>
-                      </Button>
-                    </Link>
-                  </span>
+                <div
+                  className={`flex ${
+                    passwordResetEnabled ? 'justify-between' : 'justify-end'
+                  }`}
```

Comment:
If we reverse the flex direction here, you can leave it as justify-between and don't need a conditional.

Reply:
Done!